CHANGELOG.md

```diff
@@ -38,6 +39,7 @@ All notable changes to this project will be documented in this file based on the
 - Fix php notice on `\Elastica\Index::getAliases()` if index has no aliases [#1078](https://github.com/ruflin/Elastica/issues/1078)
 
 ### Added
+- `Elastica\Aggregations\GeoCentroid`
```

Comment:
Seems like your merge now caused it to be added twice. This one should be removed.